### PR TITLE
Fix string refcounting in context.c

### PR DESCRIPTION
**Pull request by xVin**
adds missing fixes in context.c

### CODE REVIEW

File: src/context.c [hunk 1]
```diff
@@ -221,6 +223,7 @@ func_exec(
 
     while (n) {
         --n;
+        ws_object_unref((struct ws_object*) &stack[i].string);
```
Comment (matthiasbeyer) [inline src/context.c]: I'm relatively sure we don't need this.

Comment (matthiasbeyer) [inline src/context.c]: But not entirely. We should ask @neithernut .

Comment (xVin) [inline src/context.c]: Yeah wasn't sure there as well.

Comment (neithernut) [inline src/context.c]: The stack itself unrefs/has to unref the values when popping (which, in turn, the action processor does).

**Comment by xVin** [comment]
Updated.

**Comment by neithernut** [comment]
ACK.

**Pull request merged**